### PR TITLE
Update CI - faster runs, simpler set-up

### PR DESCRIPTION
**Pull request by Robinlovelace**

### CODE REVIEW

**Comment by agila5** [comment]
Thanks! I was thinking more or less the same a few days ago. I don't understand most of the changes in the .yaml file but it works 🎉

File: .github/workflows/R-CMD-check.yaml [hunk 1]
```diff
@@ -1,16 +1,19 @@
...
   pull_request:
     branches:
+      - main
       - master
 
 name: R-CMD-check
 
 jobs:
   R-CMD-check:
-    if: "!(contains(github.event.head_commit.message, 'skip ci')||contains(github.event.head_commit.message, 'ci skip')||contains(github.event.head_commit.message, 'skip-ci')||contains(github.event.head_commit.message, 'ci-skip')||contains(github.event.head_commit.message, 'skip_ci')||contains(github.event.head_commit.message, 'ci_skip'))"
```
Comment (agila5) [inline .github/workflows/R-CMD-check.yaml:13]: Just one question. Would you mind "restoring" this line? I'm not 100% sure that it still works ( will test it later) but I used that to skip the R CMD checks for the simple commits (change the docs and so on)

**Review by agila5 — APPROVED** [review]

File: .github/workflows/R-CMD-check.yaml (hunk 1 above)
Comment (Robinlovelace) [reply .github/workflows/R-CMD-check.yaml:13]: Sure, will do.

**Comment by Robinlovelace** [comment]
Read to go I think, if it passes! :crossed_fingers:

**Pull request merged**